test/models/fact_check_test.rb

```diff
@@ -596,4 +596,26 @@ def setup
       end
     end
   end
+
+  test "should not have duplicate tags" do
+    pm = create_project_media
+    cd = create_claim_description(project_media: pm)
+    fc = create_fact_check claim_description: cd, tags: ['one', 'one', '#one']
```

Comment:
also fc = create_fact_check claim_description: cd, tags: ['one', 'one', '#one', 'ONE', '#ONE'] should pass with same tags count